### PR TITLE
Add scooby

### PR DESCRIPTION
**Pull request by banesullivan**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by banesullivan** [comment]
@conda-forge/help-python any tips on why the linting failed?

**Comment by banesullivan** [comment]
@prisae - are you willing to be listed as a maintainer of this? Can you please comment to confirm or deny

**Comment by mariusvniekerk** [comment]
it looks mostly okay to me from the lint side.  try running the lint locally?

**Comment by banesullivan** [comment]
> try running the lint locally?

I'm unable to install conda smithy locally - it's been hung on the install for a really long time

**Comment by banesullivan** [comment]
From a local lint:

```bash
(base) ➜  scooby git:(master) conda smithy recipe-lint .
Traceback (most recent call last):
  File "/Users/bane/anaconda3/bin/conda-smithy", line 10, in <module>
    sys.exit(main())
  File "/Users/bane/anaconda3/lib/python3.6/site-packages/conda_smithy/cli.py", line 485, in main
    args.subcommand_func(args)
  File "/Users/bane/anaconda3/lib/python3.6/site-packages/conda_smithy/cli.py", line 372, in __call__
    return_hints=True,
  File "/Users/bane/anaconda3/lib/python3.6/site-packages/conda_smithy/lint_recipe.py", line 536, in main
    meta = ruamel.yaml.load(content, ruamel.yaml.RoundTripLoader)
  File "/Users/bane/anaconda3/lib/python3.6/site-packages/ruamel/yaml/main.py", line 935, in load
    return loader._constructor.get_single_data()
  File "/Users/bane/anaconda3/lib/python3.6/site-packages/ruamel/yaml/constructor.py", line 109, in get_single_data
    node = self.composer.get_single_node()
  File "/Users/bane/anaconda3/lib/python3.6/site-packages/ruamel/yaml/composer.py", line 78, in get_single_node
    document = self.compose_document()
  File "/Users/bane/anaconda3/lib/python3.6/site-packages/ruamel/yaml/composer.py", line 101, in compose_document
    node = self.compose_node(None, None)
  File "/Users/bane/anaconda3/lib/python3.6/site-packages/ruamel/yaml/composer.py", line 138, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/Users/bane/anaconda3/lib/python3.6/site-packages/ruamel/yaml/composer.py", line 218, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/Users/bane/anaconda3/lib/python3.6/site-packages/ruamel/yaml/composer.py", line 138, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/Users/bane/anaconda3/lib/python3.6/site-packages/ruamel/yaml/composer.py", line 211, in compose_mapping_node
    while not self.parser.check_event(MappingEndEvent):
  File "/Users/bane/anaconda3/lib/python3.6/site-packages/ruamel/yaml/parser.py", line 141, in check_event
    self.current_event = self.state()
  File "/Users/bane/anaconda3/lib/python3.6/site-packages/ruamel/yaml/parser.py", line 581, in parse_block_mapping_key
    token.start_mark,
ruamel.yaml.parser.ParserError: while parsing a block mapping
  in "<unicode string>", line 29, column 3:
      home: https://github.com/banesul ...
      ^ (line: 29)
expected <block end>, but found '<scalar>'
  in "<unicode string>", line 33, column 76:
     ...  report your Python environment's package versions and hardware  ...
                                         ^ (line: 33)
```

File: recipes/scooby/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+test:
+  imports:
+    - scooby
+
+about:
+  home: https://github.com/banesullivan/scooby
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'A lightweight toolset to easily report your Python environment's package versions and hardware resources.'
```
Comment (beckermr) [inline recipes/scooby/meta.yaml]: Yeah you have not quoted the `'` properly. Either you need to use the YAML `|` syntax or you need to have double quotes around the whole string.

Comment (banesullivan) [reply recipes/scooby/meta.yaml]: Ah thanks!

**Comment by banesullivan** [comment]
@conda-forge/help-python - I thought I did the licensing file properly? Any tips? thanks!

**Comment by beckermr** [comment]
The license file is not in the pypi sdist. You will need to add a `MANIFEST.in` to your repo and explicitly tell setuptools to package the LICENSE file in the sdist. Once you do that, cut a new version and push to pypi. From there, things should work. 

This SO should help with the formatting of the manifest file. 

https://stackoverflow.com/questions/24727709/do-python-projects-need-a-manifest-in-and-what-should-be-in-it

You may need to add the `include_package_data` keyword for setuptools in your setup.py. I am not 100% sure.

**Comment by beckermr** [comment]
The other option is to place a copy of the license directly in the recipe directory. This is a temporary workaround supported by conda, but it is better to actually ship the file.

**Comment by banesullivan** [comment]
Woot! 🎉 - all looks good to me!

**Comment by banesullivan** [comment]
@conda-forge/staged-recipes I think this is ready to go!

**Comment by beckermr** [comment]
We need affirmation from @prisae then we are good to go!

**Comment by prisae** [comment]
Yes, I am fine with that.

**Pull request merged**